src/current/_includes/molt/migration-prepare-database.md

```diff
@@ -23,4 +149,115 @@ For MySQL **5.7** sources, set the following values. Note that `binlog-row-image
 - `--binlog-row-image=full`
 - `--server-id={ID}`
 - `--log-bin=log-bin`
-</section>
\ No newline at end of file
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+##### Create source sentinel table
+
+Create a checkpoint table called `_replicator_sentinel` in the Oracle schema you will migrate:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE migration_schema."_replicator_sentinel" (
+  keycol NUMBER PRIMARY KEY,
+  lastSCN NUMBER
+);
+~~~
+
+Grant permissions to modify the checkpoint table. In Oracle Multitenant, grant the permissions on the PDB:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SELECT, INSERT, UPDATE ON migration_schema."_replicator_sentinel" TO C##MIGRATION_USER;
+~~~
+
+##### Grant LogMiner privileges
+
+Grant LogMiner privileges. In Oracle Multitenant, grant the permissions on the CDB:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+-- Access to necessary V$ views
+GRANT SELECT ON V_$LOG TO C##MIGRATION_USER;
+GRANT SELECT ON V_$LOGFILE TO C##MIGRATION_USER;
+GRANT SELECT ON V_$LOGMNR_CONTENTS TO C##MIGRATION_USER;
+GRANT SELECT ON V_$ARCHIVED_LOG TO C##MIGRATION_USER;
+GRANT SELECT ON V_$LOG_HISTORY TO C##MIGRATION_USER;
+
+-- SYS-prefixed views (for full dictionary access)
+GRANT SELECT ON SYS.V_$LOGMNR_DICTIONARY TO C##MIGRATION_USER;
+GRANT SELECT ON SYS.V_$LOGMNR_LOGS TO C##MIGRATION_USER;
+GRANT SELECT ON SYS.V_$LOGMNR_PARAMETERS TO C##MIGRATION_USER;
+GRANT SELECT ON SYS.V_$LOGMNR_SESSION TO C##MIGRATION_USER;
+
+-- Access to LogMiner views and controls
+GRANT LOGMINING TO C##MIGRATION_USER;
+GRANT EXECUTE ON DBMS_LOGMNR TO C##MIGRATION_USER;
+~~~
+
+The user is responsible for querying redo logs from LogMiner, querying active transaction information to obtain the starting point for initial ongoing replication, and updating the internal `_replicator_sentinel` table previously created on the Oracle source schema by the DBA.
+
+##### Verify LogMiner privileges
```

Comment:
Great level of detail here! I think this will help folks do a good sanity check.